app/uk/gov/hmrc/emailverification/services/AuditService.scala

```diff
@@ -249,6 +249,38 @@ class AuditService @Inject() (
     sendEvent("PasscodeVerificationResponse", details, "HMRC Gateway - Email Verification - verifies the passcode matches with the stored passcode and HMRC email address")
   }
 
+  //************ JourneyController Events **********************
+
+  def sendVerifyEmailRequestReceivedEvent(verifyEmailRequest: VerifyEmailRequest, responseStatus: Int)(implicit request: Request[_]): Future[Unit] = {
+    val details = Map(
+      "credId" -> verifyEmailRequest.credId,
+      "bearerToken" -> hc.authorization.getOrElse(Authorization("-")).value,
+      "origin" -> verifyEmailRequest.origin,
+      "continueUrl" -> verifyEmailRequest.continueUrl,
+      "deskproServiceName" -> verifyEmailRequest.deskproServiceName.getOrElse("-"),
+      "accessibilityStatementUrl" -> verifyEmailRequest.accessibilityStatementUrl,
+      "pageTitle" -> verifyEmailRequest.pageTitle.getOrElse("-"),
+      "backUrl" -> verifyEmailRequest.backUrl.getOrElse("-"),
+      "emailAddress" -> verifyEmailRequest.email.fold("-")(_.address),
+      "emailEntryUrl" -> verifyEmailRequest.email.fold("-")(_.enterUrl),
+      "lang" -> verifyEmailRequest.lang.fold("-")(_.value),
+      "status" -> responseStatus.toString
+    )
+    logger.info(s"[GG-5646] VerifyEmailRequest received. $requestContextForLog")
+    sendEvent("VerifyEmailRequest", details, "HMRC Gateway - Email Verification - a VerifyEmailRequest has been made to verify an email address belongs to the requestor.")
+  }
+
+  def sendEmailVerificationOutcomeRequestEvent(credId: String, emailsStatusJson: String, responseStatus: Int)(implicit request: Request[_]): Future[Unit] = {
+    val details = Map(
+      "credId" -> credId,
+      "bearerToken" -> hc.authorization.getOrElse(Authorization("-")).value,
+      "emails" -> emailsStatusJson,
+      "status" -> responseStatus.toString
+    )
```

Comment:
according to GG-5646  userAgentString and statusCode do not match

Reply:
thanks